packages/legend-application-marketplace/src/pages/VendorData/LegendMarketplaceVendorData.tsx

```diff
@@ -0,0 +1,202 @@
+      {radioOptions.map((option: string) => (
+        <button
+          key={option}
+          onClick={() => setSelected(option)}
+          className={clsx('legend-marketplace-radio-button', {
+            'legend-marketplace-radio-button__selected': selected === option,
+          })}
+        >
+          {option}
+        </button>
+      ))}
+    </div>
+  );
+});
+
+const SearchResultsRenderer = observer(
+  (props: { providerResults: ProviderResult[]; sectionTitle: string }) => {
+    const { providerResults, sectionTitle } = props;
+    const applicationStore = useApplicationStore();
+
+    const onAddToCartClick = (providerResult: ProviderResult) => {
+      applicationStore.notificationService.notifySuccess(
+        `'${providerResult.productName}' is added to your cart`,
+      );
+    };
+
+    return (
+      <div>
+        <div className="legend-marketplace-vendordata-main-search-results__category">
+          <div className="legend-marketplace-vendordata-main-sidebar__title">
+            {sectionTitle}
+          </div>
+          <a href="#" className="see-all">
+            <strong>See All&gt;</strong>
+          </a>
+        </div>
+        <div className="legend-marketplace-vendordata-main-search-results__card-group">
+          {providerResults.map((provider) => (
+            <LegendMarketplaceProviderCard
+              key={provider.id}
+              providerResult={provider}
+              onAddToCartClick={() => onAddToCartClick(provider)}
+            />
+          ))}
+        </div>
+      </div>
+    );
+  },
+);
+
+export const VendorDataMainContent = observer(
+  (props: { marketPlaceVendorDataState: LegendMarketPlaceVendorDataState }) => {
+    const { marketPlaceVendorDataState } = props;
+
+    return (
+      <div className="legend-marketplace-vendordata-main">
+        <div className="legend-marketplace-vendordata-main-sidebar col-sm-4">
+          <div className="legend-marketplace-vendordata-main-sidebar__title">
+            Filters
+          </div>
+          <hr></hr>
+          <div className="legend-marketplace-vendordata-main-sidebar__subtitle">
+            Providers
+          </div>
+          <div className="legend-marketplace-vendordata-main-content__sidebar__checkbox-filter-group">
+            {marketPlaceVendorDataState.dataFeedProviders.map((vendor) => (
+              <div
+                key={vendor.id}
+                className="legend-marketplace-vendordata-main-sidebar__checkbox-filter"
+              >
+                <input type="checkbox" value={vendor.providerName} />
+                <label>{vendor.providerName}</label>
+              </div>
+            ))}
+          </div>
+        </div>
+        <div className="legend-marketplace-vendordata-main-search-results">
+          <SearchResultsRenderer
+            providerResults={marketPlaceVendorDataState.dataFeedProviders}
+            sectionTitle="Data Feed"
+          />
+          <hr></hr>
```

Comment:
Nit: can replace this with
```suggestion
          <hr/>
```